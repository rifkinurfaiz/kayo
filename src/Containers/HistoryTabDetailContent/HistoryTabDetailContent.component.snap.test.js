import * as React from 'react';
import { shallow } from 'enzyme';

import HistoryTabDetailContentComponent from './HistoryTabDetailContent.component';

describe('HistoryTabDetailContentComponent', () => {
  const props = [
    {
      year: 2020,
      data: [
        {
          month: 'January',
          assets: [
            {
              id: 'jenius',
              name: 'Jenius',
              amount: 'Rp10.000.000'
            },
            {
              id: 'bca',
              name: 'BCA',
              amount: 'Rp10.200.140.234'
            },
            {
              id: 'ovo',
              name: 'OVO',
              amount: 'Rp2.040.000'
            },
            {
              id: 'gopay',
              name: 'Gopay',
              amount: 'Rp1.000'
            }
          ],
          totalAmount: 'Rp212.181.234'
        }
      ]
    }
  ];

  it('should render correctly', () => {
    expect(
      shallow(<HistoryTabDetailContentComponent data={props} />)
    ).toMatchSnapshot();
  });
});
