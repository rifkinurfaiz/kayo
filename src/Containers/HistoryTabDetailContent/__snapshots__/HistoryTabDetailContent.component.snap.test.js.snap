// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HistoryTabDetailContentComponent should render correctly 1`] = `
<View>
  <View
    style={
      Object {
        "alignItems": "center",
        "justifyContent": "center",
        "marginVertical": 13,
      }
    }
  >
    <Text
      style={
        Object {
          "fontSize": 17,
          "fontWeight": "bold",
        }
      }
    />
  </View>
  <ScrollView
    style={
      Object {
        "marginBottom": 100,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "flexWrap": "wrap",
          "justifyContent": "space-between",
          "paddingHorizontal": 13,
        }
      }
    >
      <DetailItem
        data={
          Array [
            Object {
              "accounts": Array [
                Object {
                  "amount": "Rp10.000.000",
                  "id": "jenius",
                  "name": "Jenius",
                },
                Object {
                  "amount": "Rp10.200.140.234",
                  "id": "bca",
                  "name": "BCA",
                },
                Object {
                  "amount": "Rp2.040.000",
                  "id": "ovo",
                  "name": "OVO",
                },
                Object {
                  "amount": "Rp1.000",
                  "id": "gopay",
                  "name": "Gopay",
                },
              ],
              "month": "January",
              "totalAmount": "Rp212.181.234",
            },
          ]
        }
        key="0"
        year={2020}
      />
    </View>
  </ScrollView>
</View>
`;
